subworkflows/local/normalize/main.nf

```diff
@@ -2,7 +2,7 @@
 // Perform normalizations (currently using Nacho)
 //
 
-include { NACHO_NORMALIZE } from '../../../modules/local/nacho/normalize'
+include { NACHO_NORMALIZE } from '../../../modules/local/nacho/normalize/main'
```

Comment:
Typically there is no need to add the "/main" whenever there is a main.nf file inside the folder specified, nextflow already looks for it. So you could just leave this file unmodified and will work regardless